packages/editor/src/ce/extensions/core/read-only-extensions.ts

```
@@ -1,3 +1,12 @@
 import { Extensions } from "@tiptap/core";
+// types
+import { TExtensions } from "@/types";
 
-export const CoreReadOnlyEditorAdditionalExtensions = (): Extensions => [];
+type Props = {
+  disabledExtensions: TExtensions[];
+};
+
+export const CoreReadOnlyEditorAdditionalExtensions = (props: Props): Extensions => {
+  const {} = props;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)